layouts/shortcodes/python-venv.html

```diff
@@ -0,0 +1,24 @@
+<p>
+    For Python, before we deploy the stack, the following commands need to be
+    run to create a virtual environment, activate it, and install dependencies:
+</p>
+
+<p>Create a virtual environment:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ python3 -m venv venv</code></pre>
+</div>
+
+<p>Activate the environment in Linux and macOS:</p>
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ source venv/bin/activate</code></pre>
+</div>
+
+<p>Activate the environment in Windows:</p>
```

Comment:
Alternatively, we could use the OS chooser here...

I realize there's duplication for mac/linux, but this allows using the right `python` vs `python3` command on Windows, and allows using the right `>` vs `$` prompts that we've started using elsewhere.

```html
{{/*
This table is effectively the same as oschoose.md, however it
needs to be embedded inline because shortcodes cannot reference
other shortcodes.
*/}}
<div class="mdl-tabs os-tab-tabs">
    <div class="mdl-tabs__tab-dir os-tab-tabstrip">
        <a data-track="oschoose-macOs" class="mdl-tabs__tab os-tab is-active">macOS</a>
        <a data-track="oschoose-linux" class="mdl-tabs__tab os-tab">Linux</a>
        <a data-track="oschoose-windows" class="mdl-tabs__tab os-tab">Windows</a>
    </div>
</div>

{{/*
.mdl-tabs__tab uses floats, so we need to use clearfix to avoid
wrapping/indenting the subsequent element containing the code snippet.
*/}}
<div class="clearfix"></div>

<div class="os-prologue-macos"></div>
<div>
    <p>Create a virtual environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ python3 -m venv venv</code></pre>
    </div>

    <p>Activate the environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ source venv/bin/activate</code></pre>
    </div>

    <p>Install dependencies:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ pip3 install -r requirements.txt</code></pre>
    </div>
</div>

<div class="os-prologue-linux"></div>
<div>
    <p>Create a virtual environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ python3 -m venv venv</code></pre>
    </div>

    <p>Activate the environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ source venv/bin/activate</code></pre>
    </div>

    <p>Install dependencies:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bash" data-lang="bash">$ pip3 install -r requirements.txt</code></pre>
    </div>
</div>

<div class="os-prologue-windows"></div>
<div>
    <p>Create a virtual environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bat" data-lang="bat"><span class="p">&gt;</span> python -m venv venv</code></pre>
    </div>

    <p>Activate the environment:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bat" data-lang="bat"><span class="p">&gt;</span> venv\Scripts\activate</code></pre>
    </div>

    <p>Install dependencies:</p>
    <div class="highlight">
        <pre class="chroma"><code class="language-bat" data-lang="bat"><span class="p">&gt;</span> pip3 install -r requirements.txt</code></pre>
    </div>
</div>
```

Reply:
Great suggestion!